src/blocking/i2c.rs

```diff
@@ -127,3 +127,26 @@ pub trait WriteIterRead {
     where
         B: IntoIterator<Item = u8>;
 }
+
+/// Actions for transactional I2C trait
+///
+/// This allows composition of I2C operations into a single bus transaction
+#[cfg(feature = "unproven")]
+#[derive(Debug, PartialEq)]
+pub enum Actions<'a> {
```

Comment:
Enums usually have a singular name, so `Action`. Also, why not `Operation`?

Reply:
Yeah really good point, I'm not sure why I didn't just use `Transaction` tbh, does that work for you?

Reply:
Sounds good!